extensions/schema-compare/src/dialogs/schemaCompareDialog.ts

```diff
@@ -720,6 +717,7 @@ export class SchemaCompareDialog {
 			});
 		}
 
+		this.dialog.okButton.enabled = await this.shouldEnableOkayButton();
```

Comment:
Just to confirm - this will work in both dacpac and database cases - yeah?

Reply:
Yup, the logic for when the dacpac radio button is getting selected is unchanged, and still works in my clicking around and launching from different points.